events/story_cycles/wc_story_cycle_horde_invasion_events.txt

```diff
@@ -98,13 +86,50 @@ WCHOI.3 = {
 			}
 		}
 		name = WCHOI_3_OPT_A
+
+		if = {
+			limit = {
+				NOT = { root = scope:warchief }
+				highest_held_title_tier < scope:warchief.highest_held_title_tier
+			}
+
+			# offer_fealty_interaction
+			scope:warchief = {
+				save_scope_as = recipient
+				add_opinion = {
+					target = root
+					modifier = grateful_opinion
+					opinion = 30
+				}
+			}
+			root = { save_scope_as = actor }
+			offer_fealty_interaction_effect = yes
+			trigger_event = char_interaction.0010
+		}
+		else = {
+			add_character_modifier = {
+				modifier = attempting_reunification_modifier
+				years = 3
+			}
```

Comment:
I should remind that it's a player-only event, so the former AI warchief can't get prestige or this modifier.

Reply:
Reworked it - tell what you think